### ISSUE TITLE
NativeClass is not defined

### ISSUE THREAD

**Issue by blasus** [Notalib/nativescript-accessibility-ext#58]
After updating NativeScript the plugin is crashing the app. This error is common when ts-patch is not being ran properly for a package build replacing the NativeClass decorator. The latest version 7.0.2 of @nota/nativescript-accessibility-ext seems to have this issue.

### Versions ###
```
✔ Component nativescript has 7.1.2 version and is up to date.
✔ Component @nativescript/core has 7.3.0 version and is up to date.
✔ Component @nativescript/android has  7.0.1 version and is up to date.
✔ Component @nativescript/ios has 7.2.0 version and is up to date.

@nota/nativescript-accessibility-ext@7.0.2
```
### Crash ###
```
JS: Angular is running in development mode. Call enableProdMode() to enable production mode.

System.err: An uncaught Exception occurred on "main" thread.

System.err: Calling js method onStart failed

System.err: ReferenceError: NativeClass is not defined

System.err: 

System.err: StackTrace:

System.err: ensureNativeClasses(file: node_modules\@nota\nativescript-accessibility-ext\utils\accessibility-helper.android.js:266:0)

System.err: 	at setAccessibilityDelegate(file: node_modules\@nota\nativescript-accessibility-ext\utils\accessibility-helper.android.js:462:0)

System.err: 	at updateAccessibilityProperties(file: node_modules\@nota\nativescript-accessibility-ext\utils\accessibility-helper.android.js:380:0)

System.err: 	at accessibleSetNative(file: node_modules\@nota\nativescript-accessibility-ext\ui\core\view.android.js:75:23)

System.err: 	at applyAllNativeSetters(file: node_modules\@nativescript\core\ui\core\properties\index.js:1153:0)

System.err: 	at initNativeView(file: node_modules\@nativescript\core\ui\core\properties\index.js:1080:0)

System.err: 	at onResumeNativeUpdates(file: node_modules\@nativescript\core\ui\core\view-base\index.js:695:22)

System.err: 	at _resumeNativeUpdates(file: node_modules\@nativescript\core\ui\core\view-base\index.js:259:0)

System.err: 	at onLoaded(file: node_modules\@nativescript\core\ui\core\view-base\index.js:214:0)

System.err: 	at onLoaded(file: node_modules\@nativescript\core\ui\core\view\index.android.js:283:0)

System.err: 	at (file: node_modules\@nativescript\core\ui\core\view-base\index.js:297:0)

System.err: 	at callFunctionWithSuper(file: node_modules\@nativescript\core\ui\core\view-base\index.js:291:0)

System.err: 	at callLoaded(file: node_modules\@nativescript\core\ui\core\view-base\index.js:297:0)

System.err: 	at onStart(file: node_modules\@nativescript\core\ui\frame\index.android.js:985:0)

System.err: 	at onStart(file: node_modules\@nativescript\core\ui\frame\activity.android.js:31:0)

System.err: 	at com.tns.Runtime.callJSMethodNative(Native Method)

System.err: 	at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1302)

System.err: 	at com.tns.Runtime.callJSMethodImpl(Runtime.java:1188)

System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:1175)

System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:1153)

System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:1149)

System.err: 	at com.tns.NativeScriptActivity.onStart(NativeScriptActivity.java:39)

System.err: 	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1425)

System.err: 	at android.app.Activity.performStart(Activity.java:7825)

System.err: 	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3294)

System.err: 	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:221)

System.err: 	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:201)

System.err: 	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:173)

System.err: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)

System.err: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)

System.err: 	at android.os.Handler.dispatchMessage(Handler.java:107)

System.err: 	at android.os.Looper.loop(Looper.java:214)

System.err: 	at android.app.ActivityThread.main(ActivityThread.java:7356)

System.err: 	at java.lang.reflect.Method.invoke(Native Method)

System.err: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)

System.err: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)

JS: 2021-03-08 13:18:23 [ERROR] Uncaught exception: Error: Calling js method onStart failed

JS: ReferenceError: NativeClass is not defined

JS: 2021-03-08 13:18:23 [ERROR] ensureNativeClasses(file: node_modules\@nota\nativescript-accessibility-ext\utils\accessibility-helper.android.js:266:0)

JS: 	at setAccessibilityDelegate(file: node_modules\@nota\nativescript-accessibility-ext\utils\accessibility-helper.android.js:462:0)

JS: 	at updateAccessibilityProperties(file: node_modules\@nota\nativescript-accessibility-ext\utils\accessibility-helper.android.js:380:0)

JS: 	at accessibleSetNative(file: node_modules\@nota\nativescript-accessibility-ext\ui\core\view.android.js:75:23)

JS: 	at applyAllNativeSetters(file: node_modules\@nativescript\core\ui\core\properties\index.js:1153:0)

JS: 	at initNativeView(file: node_modules\@nativescript\core\ui\core\properties\index.js:1080:0)

JS: 	at onResumeNativeUpdates(file: node_modules\@nativescript\core\ui\core\view-base\index.js:695:22)

JS: 	at _resumeNativeUpdates(file: node_modules\@nativescript\core\ui\core\view-base\index.js:259:0)

JS: 	at onLoaded(file: node_modules\@nativescript\core\ui\core\view-base\index.js:214:0)

JS: 	at onLoaded(file: node_modules\@nativescript\core\ui\core\view\index.android.js:283:0)

JS: 	at (file:/...
```

### Devices ###
Tested on Android emulator - Pixel 2 (API 29), v. 10

**Comment by lambourn** [comment]
same issue here. Any ideas how to fix it?

**Comment by m-abs** [comment]
Try v7.0.3.

**Comment by lambourn** [comment]
thx @m-abs will do!

**Comment by lambourn** [comment]
@m-abs I can confirm, issue is resolved with 7.0.3.

**Comment by blasus** [comment]
I can confirm too, version 7.0.3 tested on both android and ios.
